Sources/Networking/Responses/WebOfferingProductsResponse.swift

```diff
@@ -15,43 +15,10 @@ import Foundation
 
 struct WebOfferingProductsResponse {
 
-    struct Price {
-        let amountMicros: Int64
-        // This will be a 3-letter currency code
-        let currency: String
-    }
-
-    struct PricingPhase {
-        let periodDuration: String?
-        let price: Price?
-        let cycleCount: Int
-    }
-
-    struct PurchaseOption {
-        // Only for non-subscriptions
-        @IgnoreDecodeErrors<Price?>
-        var basePrice: Price?
-
-        // Only for subscriptions
-        @IgnoreDecodeErrors<PricingPhase?>
-        var base: PricingPhase?
-        @IgnoreDecodeErrors<PricingPhase?>
-        var trial: PricingPhase?
-    }
-
-    struct Product {
-        let identifier: String
-        let productType: String
-        let title: String
-        let description: String?
-        let defaultPurchaseOptionId: String?
-        let purchaseOptions: [String: PurchaseOption]
-    }
```

Comment:
These are now part of `WebProductsResponse`, since the models are exactly the same